packages/mqttsuite/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/SNodeC/mqttsuite
+TERMUX_PKG_DESCRIPTION="MQTTSuite: A lightweight MQTT Integration System"
+TERMUX_PKG_LICENSE="MIT OR LGPL-3.0-or-later"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
```

Comment:
```suggestion
```

The license file only needs to be specified if the build system is unable to find it on in the "common license files" list.
https://github.com/termux/termux-packages/blob/054de3f8fe10c4f51a1a518142fd0981099c704f/scripts/build/termux_step_install_license.sh#L36-L44